site/_guides/cert-manager.md

```diff
@@ -487,11 +505,108 @@ $ curl https://httpbin.davecheney.com/get
+Once that's done, create the dummy Ingress object:
+
+```yaml
+apiVersion: networking.k8s.io/v1beta1
+kind: Ingress
+metadata:
+  annotations:
+    cert-manager.io/cluster-issuer: letsencrypt-prod
+    ingress.kubernetes.io/force-ssl-redirect: "true"
+    kubernetes.io/ingress.allow-http: "false"
+    kubernetes.io/ingress.class: contour
+    kubernetes.io/tls-acme: "true"
+  name: httpbinproxy
+  namespace: default
+spec:
+  rules:
+  - host: httpbinproxy.davecheney.com
+    http:
+      paths:
+      - backend:
+          serviceName: httpbin
+          servicePort: 8080
+  tls:
+  - hosts:
+    - httpbinproxy.davecheney.com
+    secretName: httpbinproxy
+```
+
+Once cert-manager has done its thing, you will have a `kubernetes.io/tls` Secret present, that will contain the keypair.
+Contour will detect that the Secret exists and generate the HTTPProxy config.
+
+After that, you should be able to curl the new site:
+
+```sh
+$ curl https://httpbinproxy.davecheney.com/get
+{
+  "args": {},
+  "headers": {
+    "Accept": "*/*",
+    "Content-Length": "0",
+    "Host": "httpbinproxy.davecheney.com",
+    "User-Agent": "curl/7.54.0",
+    "X-Envoy-Expected-Rq-Timeout-Ms": "15000",
+    "X-Envoy-External-Address": "122.106.57.183"
+  },
+  "origin": "122.106.57.183",
+  "url": "https://httpbinproxy.davecheney.com/get"
+}
+```
+
+### Caveats
+
+This method is a workaround until we can deliver the changes in [#950](https://github.com/projectcontour/contour/issues/950) and [#951](https://github.com/projectcontour/contour/issues/951).
+As such, it has a few caveats.
+
+Firstly, it works because Contour will override matching FQDN/path combinations in the following order: HTTPProxy overrides IngressRoute overrides Ingress.
+The solver Ingress created by cert-manager has a unique FQDN/path combination so it will work, but the base one will be overridden by the HTTPProxy config.
+
+Secondly, the actual config of where the object sends traffic is irrelevant, *as long as it matches a route in the HTTPProxy*.
+In this example, we're not using any advanced routing features, so this is straightforward.
+However, if you are, you'll either need to match routes between Ingress and HTTPProxy or create a dummy path for the Ingress to match on.
+
 ## Wrapping up
 
 Now that you've deployed your first HTTPS site using Contour and Let's Encrypt, deploying additional TLS enabled services is much simpler.
-Remember that for each HTTPS website you deploy, you create a Certificate CRD that provides the domain name and the name of the target Secret.
-After the Secret is created, you add a `tls` field to the Ingress object for your site.
+Remember that for each HTTPS website you deploy, cert-manager will  create a Certificate CRD that provides the domain name and the name of the target Secret.
+After the Secret is created, you add a `tls` field to the HTTPProxy or Ingress object for your site.
```

Comment:
The order of operations isn't this strict, contour is just not going to be able to serve the https version of the site until the secret is ready.

Reply:
Okay, I'll write it differently, thanks.